SHIELD-GLOSSARY.md

```diff
@@ -0,0 +1,95 @@
+# Shield Glossary
+
+## Introduction
+
+In the greater [Shield project](https://wiki.mozilla.org/Firefox/Shield),
+with the code projects [Normandy](https://github.com/mozilla/normandy) and
+[Delivery Console](https://github.com/mozilla/delivery-console/) in mind,
+there are a lot of different terms to remember and understand.
+
+Here we try
+to list them with the goal of...:
+
+1. Getting everyone to use the same terms
+
+2. Understanding what they are and how they're different to each other
+
+3. Make it easier to people new to the using the projects and people
+   coding the code for the services
+
+
+
+## Glossary
+
+* **Addon Studies**
+
+    * When you first install an extension, then a recipe that sends the data to supply the extension to do a study.
+    * *Alias:* Opt Out Studies
```

Comment:
This isn't right. The extension isn't installed before hand, the recipe includes the extension to install. Additionally, Normandy doesn't send any additional data for the study, the add-on is fully self contained (at least right now). Here's an example of a add-on study recipe. Note that the there is a link to the add-on here in the recipe.

```json
{
  "id": 300,
    "name": "Opt-Out Ongoing Satisfaction on Release",
    "revision_id": "1670",
    "action": "opt-out-study",
    "arguments": {
      "name": "56-satisfaction-1406537",
      "description": "Satisfaction Study: https://bugzilla.mozilla.org/show_bug.cgi?id=1406537",
      "addonUrl": "https://net-mozaws-prod-us-west-2-normandy.s3.amazonaws.com/extensions/57-perception-shield-studymozilla.com-1.0.2-signed.xpi",
      "isEnrollmentPaused": true
    },
    "filter_expression": "normandy.locale in ['en-US', 'en-AU', 'en-CA', 'en-GB', 'en-NZ', 'en-ZA'] && normandy.version >=  '56.0' && normandy.channel == 'release' && [normandy.userId, normandy.recipeId]|bucketSample((normandy.request_time / (24*60*60*1000)), 1, 1000))"
}
```